src/Stripe.net/Services/Issuing/Authorizations/AuthorizationService.cs

```diff
@@ -23,54 +21,56 @@ public AuthorizationService(string apiKey)
         {
         }
 
+        public override string ObjectName => "issuing.authorization";
+
         public virtual Authorization Approve(string authorizationId, AuthorizationApproveOptions options = null, RequestOptions requestOptions = null)
         {
-            return this.Post($"{classUrl}/{authorizationId}/approve", requestOptions, options);
+            return this.PostRequest<Authorization>($"{this.InstanceUrl(authorizationId)}/approve", options, requestOptions);
         }
 
         public virtual Task<Authorization> ApproveAsync(string authorizationId, AuthorizationApproveOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.PostAsync($"{classUrl}/{authorizationId}/approve", requestOptions, cancellationToken, options);
+            return this.PostRequestAsync<Authorization>($"{this.InstanceUrl(authorizationId)}/approve", options, requestOptions, cancellationToken);
```

Comment:
I wonder if it'd be cleaner to pass the authorization id + append name? Not sure what that would look like but I fell like it could be cleaner? Sorry it's a bit vague and likely not possible but I was thinking of `this.Url(authorizationId, "/approve")` or similar?

Reply:
Hm, not sure it's worth it. `this.Url(authorizationId, "/approve")` would be the same thing as `this.Url(authorizationId) + "/approve"`, which is already the same thing as `$"{this.InstanceUrl(authorizationId)}/approve"`.

Reply:
yeah fair, it's not the same approach in my head (and the former is easier to mess up). I ask because I made a mistake a few times yesterday with the /approve, /decline and all where stripe-mock still just worked because params were identical and I had messed up my URL calculation